lib/archethic/transaction_chain.ex

```diff
@@ -836,7 +836,7 @@ defmodule Archethic.TransactionChain do
     end
   end
 
-  defp fetch_genesis_address_remotely(address) when is_binary(address) do
+  def fetch_genesis_address_remotely(address) when is_binary(address) do
```

Comment:
Can you add some doc and typespec as well (as it's public now) ?

Reply:
Okay